### ISSUE TITLE
MsBot: Add deprecation warning statement in prep for sunsetting of the tool

### ISSUE THREAD

**Issue by scheyal** [microsoft/botbuilder-tools#1097]
## Describe the bug
MsBot is about to get deprecated, and replaced with common Azure practices (e.g. az cli, ARM).  We need to add a corresponding message to that effect.

###Sample Message
```
MsBot and .bot files are going to be deprecated in the next release. To provision bots, please use az cli and standard platform configuration files (e.g. appsettings or .env). 
```

**Comment by johnwildes** [comment]
Wow any timing on this?  By next release you mean next AZ CLI release?

**Comment by Diruuo** [comment]
Hey everyone, is there any update on this? 

There is still no indication that msbot is being deprecated, the only documentation that mentions .bot file deprecation is incomplete and not particularly useful (https://docs.microsoft.com/en-us/azure/bot-service/bot-file-basics?view=azure-bot-service-4.0&tabs=csharp). 

Related: MicrosoftDocs/bot-docs/issues/1150

**Comment by sgellock** [comment]
@johnwildes @Diruuo let me set some context regarding msbot and how we think about our CLI tooling strategy.  We started in our 4.3 release and continued in our 4.4 release to embrace existing Azure tooling in the areas of provisioning and deployment.  In the 4.4 release we fully embraced ARM templates for provisioning, an Azure-cli for control plane provisioning where possible.  There are many features we get moving off of recipe files and msbot clone services and onto ARM and Az-cli for provisioning.  

Additionally, msbot created and/or manipulated .bot files for storing configuration.  Again, we've moved away from this to embrace platform specific/existing mechanisms.  For C# bots, appsettings.json, for JS bots, .env files.  We received a lot of customer feedback that this was what was expected and desired.

Lastly, there was the provisioning of non-Azure services, specifically LUIS.  In this case, we have the Luis-api CLI that has always provided a CLI option for LUIS.  msbot actually made calls to this CLI under the covers.

Stepping back a bit, we have a number of CLI tools that we feel complication the experience for developers.  In our 4.5 release (our next scheduled release), we will be releasing a single CLI tool, bot Framework CLI, that is modeled in much the same way as the Azure CLI (command groups/commands/plugins).  When we are done with this effort we will have two CLI tools to consider.  Azure-CLI for Azure resources, and BF-CLI for Bot Framework (non azure) issues.  We hope this simplifies things for the community.
